pkg/synopsysctl/cmd_create.go

```diff
@@ -250,6 +276,7 @@ var createBlackDuckCmd = &cobra.Command{
 			cmd.Help()
 			return fmt.Errorf("this command takes 1 argument")
 		}
+		checkPasswords(cmd)
```

Comment:
Instead of cmd, can you pls pass only the flags

Reply:
done